### PR TITLE
📊 emissions: update carbon prices

### PR DESCRIPTION
**Pull request by pabloarosado**
This PR has many files, but the vast majority are simply moved to the archived or deleted.
* Update datasets on carbon taxes and emissions trading systems.
* Simplify code and adjust to new ETL practices and metadata.
* Minor improvements in Met Office steps.
* Archive old steps, update the dag, and remove old walden ingest scripts.
* Minor change in `tables` to avoid spurious warnings.

### CODE REVIEW

File: etl/steps/data/garden/rff/2023-10-19/emissions_weighted_carbon_price.meta.yml [hunk 1]
```diff
@@ -1,137 +1,144 @@
...
   emissions_weighted_carbon_price:
     title: Emissions-weighted carbon price
     variables:
       co2_with_ets_as_share_of_co2:
-        title: CO2 emissions covered by an ETS as a share of the country's CO2 emissions
+        title: CO₂ emissions covered by an ETS as a share of the country's CO₂ emissions
         unit: '%'
         short_unit: '%'
         display:
           numDecimalPlaces: 2
```
[the remark this answers is not in the document]
Comment (paarriagadap) [reply etl/steps/data/garden/rff/2023-10-19/emissions_weighted_carbon_price.meta.yml:30]: You could also define it in `common`

**Review by paarriagadap — APPROVED** [review]
Very good! Looks like new. I don't have much to add, except for the minor message below

File: snapshots/rff/2023-10-19/emissions_weighted_carbon_price.zip.dvc [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Emissions-weighted Carbon Price
+    description: |-
+      The Emissions-weighted Carbon Price (ECP) is an economy-wide average price on CO2 emissions. It is calculated from sector-fuel level data, which is aggregated back to the economy level using the share of each sector-fuel CO2 emissions in total GHG emissions as weights. The full methodology is described in the Resources for the Future Working Paper 22-6 [Emissions-weighted carbon price: Sources and Methods](https://www.rff.org/publications/working-papers/emissions-weighted-carbon-price-sources-and-methods/).
```
Comment (paarriagadap) [inline snapshots/rff/2023-10-19/emissions_weighted_carbon_price.zip.dvc:8]: CO₂

File: etl/steps/data/garden/rff/2023-10-19/emissions_weighted_carbon_price.meta.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pabloarosado) [reply etl/steps/data/garden/rff/2023-10-19/emissions_weighted_carbon_price.meta.yml:30]: Some of the variables include many points where the data is <1%, and it helps to visualize two decimals. As usual, the right solution would be to round to significant figures, not to decimals, but currently we can't. So, for consistency, I'll leave 2 decimals.

**Pull request merged**